GOVERNANCE.md

```diff
@@ -2,94 +2,171 @@
+
+Some contributors will not reach this level, and that's okay! L2 Maintainers still have significant responsibility and privileges in our community.
+
+#### Privileges
+
+- All privileges of the [Maintainer role](#level-2---maintainer), plus...
+- All of the privileges of L2, including...
+- `@core` role on [Discord](https://astro.build/chat)
+- New name color on Discord: **deep, dark blue**.
+- Invitation to the private #core channel on Discord.
+- Invitation to the `core` team on GitHub.
+- Ability to merge all GitHub PRs.
+- Ability to vote on all initiatives (see [Voting](#voting) below).
+
+#### Responsibilities
+
+- All of the responsibilities of L2, including...
+- Ownership over specific parts of the project.
+- Maintaining and improving overall architecture.
+- Tracking and ensuring progress of open pull requests.
+- Reviewing and merging larger, non-trivial PRs.
+
+#### Nomination
+
+To be nominated, a nominee is expected to already be performing some of the responsibilities of a Core Maintainer. This could include showing expertise over some larger section of the codebase, championing RFCs through ideation and implementation, reviewing non-trivial PRs and providing critical feedback, or some combination of those responsibilities listed above.
+
+If a Core Maintainer steps away from the project for a significant amount of time, they may be removed as a Core Maintainer (L3 -> L2) until they choose to return.
+
+In some rare cases, this role may be revoked by a project Steward. However, under normal circumstances this role is granted for as long as the contributor wishes to engage with the project.
+
+#### Nomination Process
+
+- You can be nominated by any existing Core Maintainer (L3 or above).
+- Once nominated, there will be a vote by existing Core Maintainers (L3 and above).
+- See [vote rules & requirements](#voting) for info on how the vote works.
 
-A Contributor is invited to become a Core Contributor by existing Core Contributors. A nomination will result in discussion and then a decision by the project steward(s).
 
 ### Steward
 
-Steward is an additional privilege bestowed to 1 (or more) Contributors. The role of Steward is mainly an administrative one. Stewards control and maintain sensitive project assets, and act as tiebreakers in the event of disagreements. These additional privileges include:
+Steward is an additional privilege bestowed to 1 (or more) Core Maintainers. The role of Steward is mainly an administrative one. Stewards control and maintain sensitive project assets, and act as tiebreakers in the event of disagreements.
+
+In extremely rare cases, a Steward can act unilaterally when they believe it is in the project's best interest and can prove that the issue cannot be resolved through normal governance procedure. The steward must publicly state their reason for unilateral action before taking it.
```

Comment:
> In extremely rare cases, a Steward can act unilaterally when they believe it is in the project's best interest and can prove that the issue cannot be resolved through normal governance procedure. The steward must publicly state their reason for unilateral action before taking it.

I understand the intent behind this paragraph, and I think it’s a realistic depiction of administrator privileges, but it sounds weird to put into writing. Obviously under the current stewardship I think this doesn’t matter, but I’m imagining if a different (malevolant) steward were in charge how this could be abused.

Is this worth stating that a steward can override everyone if they just announce they’re overriding everyone? Maybe they could, but I think there’d be more precedent for them to do so just from this paragraph existing.

Reply:
> if they just announce they’re overriding everyone?

At this early stage, I don't want to lose our ability to move quickly. I don't trust myself to have gotten everything right here, but I do trust that we will always have someone in the project who can be trusted to step into this role. 

If it's a dealbreaker for you, we could add a clause that states that this priviledge expires in X years, or if I ever step down as steward.

Reply:
Ah I agree with that. I read “cannot be resolved through normal governance procedure” as “people are being difficult and I don’t want to deal with them”